### PR TITLE
Replace Jekyll highlight tags with fenced code blocks

### PR DESCRIPTION
**Pull request by juliusrickert**
Replace Jekyll highlight tags with fenced code blocks.
Canonicalize javascript to js.
Correct highlighting languages.
Add highlighting to code blocks.
Remove leading $ from bash blocks.
Remove empty lines at start and end of code blocks.
Stripped trailing whitespace.

### CODE REVIEW

File: docs/_functions/domain/IMPORT_TRANSFORM.md [hunk 1]
```diff
@@ -24,8 +24,8 @@ be very error prone. Therefore instead you maintain foo.com and
 let `IMPORT_TRANSFORM` automatically generate bar.com.
 
 {% include startExample.html %}
-{% highlight html %}
 
+```html
```
Comment (tlimoncelli) [inline docs/_functions/domain/IMPORT_TRANSFORM.md:28]:
```suggestion
```text
```

File: docs/_providers/msdns.md [hunk 2]
```diff
@@ -41,25 +41,25 @@ The `ActiveDirectory_PS` provider reads an `computername` setting from
 `creds.json` to know the name of the ActiveDirectory DNS Server to run the commands on.
 Otherwise
 
-{% highlight javascript %}
+```js
```
Comment (tlimoncelli) [inline docs/_providers/msdns.md:44]:
```suggestion
```json
```

File: docs/_providers/msdns.md [hunk 3]
```diff
@@ -76,7 +76,7 @@ If you were using the `ACTIVEDIRECTORY_PS` provider and are switching to `MSDNS`
 
 During the transition your `creds.json` file might look like:
 
-{% highlight javascript %}
+```js
```
Comment (tlimoncelli) [inline docs/_providers/msdns.md:79]:
```suggestion
```json
```

**Comment by juliusrickert** [comment]
Thank you for checking this big PR so quickly :D

I'm sure there must still be some code blocks with wrong highlighting. Tried my best to fix them where ever I spotted them :D

**Pull request merged**